RFS/src/main/java/com/rfs/version_os_2_11/GlobalMetadataCreator_OS_2_11.java

```diff
@@ -91,7 +99,8 @@ protected void createComponentTemplates(GlobalMetadataData_OS_2_11 globalMetadat
 
                 logger.info("Setting Component Template: " + templateName);
                 ObjectNode settings = (ObjectNode) templates.get(templateName);
-                client.createComponentTemplate(templateName, settings);
+                client.createComponentTemplate(templateName, settings,
+                        context.createComponentTemplateContext()); // purposefully make a new one each loop iteration
```

Comment:
What do you think of rewriting with try with resource pattern rather than a commment? E.g:

```java
for (String templateName : templateWhitelist) {
    // ...
    try (final var context = context.createComponentTemplateContext()) {
        client.createComponentTemplate(templateName, settings, context);
    } 
}
```

Reply:
After revisiting I would recommend an alternative to make the method `createComponentTemplate(...)`  responsiblity to create the context